DER/CreditDerivatives/CreditDefaultSwaps.rdf

```diff
@@ -329,22 +369,21 @@
 		<skos:definition xml:lang="en">date on which credit protection is due to expire as agreed by both parties</skos:definition>
 	</owl:ObjectProperty>
 	
-	<owl:DatatypeProperty rdf:about="&fibo-der-cr-cds;sixtyBusinessDaySettlementCap">
-		<rdfs:label xml:lang="en">sixty business day settlement cap</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-fbc-fi-stl;PhysicalSettlementTerms"/>
-		<rdfs:range rdf:resource="&xsd;boolean"/>
-		<skos:definition xml:lang="en">Whether the legal terms set forth in the following language are deemed part of the confirmation (the section references are to the 2003 ISDA Credit Derivatives Definitions): &apos;Notwithstanding Section 1.7 or any provisions of Sections 9.9 or 9.10 to the contrary, but without prejudice to Section 9.3 and (where applicable) Sections 9.4, 9.5 and 9.6, if the Termination Date has not occurred on or prior to the date that is 60 Business Days following the Physical Settlement Date, such 60th Business Day shall be deemed to be the Termination Date with respect to this Transaction except in relation to any portion of the Transaction (an &quot;Affected Portion&quot;) in respect of which: (1) a valid notice of Buy-in Price has been delivered that is effective fewer than three Business Days prior to such 60th Business Day, in which case the Termination Date for that Affected Portion shall be the third Business Day following the date on which such notice is effective; or (2) Buyer has purchased but not Delivered Deliverable Obligations validly specified by Seller pursuant to Section 9.10(b), in which case the Termination Date for that Affected Portion shall be the tenth Business Day following the date on which Seller validly specified such Deliverable Obligations to Buyer.&apos;</skos:definition>
-	</owl:DatatypeProperty>
-	
 	<owl:Class rdf:about="&fibo-fbc-dae-cre;ObligationSpecificCreditEvent">
 		<rdfs:subClassOf rdf:resource="&fibo-der-cr-cds;TriggeringEvent"/>
 	</owl:Class>
 	
+	<owl:NamedIndividual rdf:about="&fibo-fbc-fi-ip;AuctionMethod">
+		<rdf:type rdf:resource="&fibo-fbc-fi-ip;PriceDeterminationMethod"/>
+		<rdfs:label xml:lang="en">auction method</rdfs:label>
+		<skos:definition xml:lang="en">method for determining a price that represents use of an independently administered synthetic auction in order to set a price</skos:definition>
```

Comment:
wording is a bit clumsy: "in order to set a price" is redundant since the clause starts "for determining a price"

Reply:
@rivettp Agreed - added DER-140 for this.